pkg/project/registry/projectrequest/delegated/sample_template.go

```diff
@@ -23,11 +25,12 @@ var (
 
 func DefaultTemplate() *templateapi.Template {
 	ret := &templateapi.Template{}
-	ret.Name = "project-request"
-	ret.Namespace = kapi.NamespaceDefault
```

Comment:
pointing out I removed the default namespace, in case you missed it... project requests still worked, and I didn't think I wanted a namespace in something we might pipe to `osc create -f -`